src/main/java/net/atos/zac/util/indexRewriteFilter.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.util;
+
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.annotation.WebFilter;
+import javax.servlet.http.HttpServletRequest;
+
+@WebFilter(filterName = "indexRewriteFilter")
+public class indexRewriteFilter implements Filter {
```

Comment:
Voeg even een stukje commentaar toe boven de class waar deze class voor bedoeld is.